frontend/www/js/omegaup/lang.en.json

```
@@ -1539,7 +1539,7 @@
 	"wordsSearchBy": "Search by",
 	"wordsSeeCourse": "See course",
 	"wordsSeeGeneralRanking": "See the general ranking",
-	"wordsSeeLatestSubmissions": "See latest submissions sent by this user",
+	"wordsSeeLatestSubmissions": "See latest submissions sent by",
```

```suggestion
	"wordsSeeLatestSubmissions": "See latest submissions sent by %(username)",
```

en general hay que evitar concatenar cadenas de traducción con otras cadenas. en otros lenguajes, el orden de las palabras puede no coincidir con el que usamos normalmente. luego en el `.vue` hay que hacer

```typescript
ui.formatString(T.wordsSeeLatestSubmissions, { username })
```